plugins/woocommerce-admin/client/marketplace/components/products/products.tsx

```diff
@@ -167,11 +167,6 @@ export default function Products( props: ProductsProps ) {
 
 	return (
 		<div className={ containerClassName }>
-			{ selectedTab === 'search' && (
```

Comment:
We should also remove the `selectedTab` import from the marketplace context, as well as the `title` and `productListTitleClassName` consts, as this is the only place they were being used.

Reply:
Nice! There were a bunch of defined but unused stuff and I did some neat cleanup here in a10eb02a40